internal/pkg/mapper/template/metadata/local.go

```diff
@@ -2,15 +2,13 @@ package metadata
 
 import (
 	"github.com/keboola/keboola-as-code/internal/pkg/model"
-	templateMetadata "github.com/keboola/keboola-as-code/internal/pkg/template/metadata"
 )
 
 func (m *metadataMapper) MapAfterLocalLoad(recipe *model.LocalLoadRecipe) error {
 	if config, ok := recipe.Object.(*model.Config); ok {
-		metadata := templateMetadata.ConfigMetadata(config.Metadata)
-		metadata.SetRepository(m.templateRef.Repository().Name)
-		metadata.SetTemplateId(m.templateRef.TemplateId())
-		metadata.SetInstanceId(m.instanceId)
+		config.Metadata.SetRepository(m.templateRef.Repository().Name)
+		config.Metadata.SetTemplateId(m.templateRef.TemplateId())
+		config.Metadata.SetInstanceId(m.instanceId)
```

Comment:
Mozno by sa to mohlo nastavovat jednou metodou, napr. `SetTemplateInstance(repo, tmpl, inst)`. Samostatne to nema velmi zmysel (ale to tak uz bolo).

Reply:
jo 👍 upravím